paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/GooglePayDivider.kt

```diff
@@ -23,23 +22,21 @@ import com.stripe.android.ui.core.shouldUseDarkDynamicColor
 internal fun GooglePayDividerUi(
     text: String = stringResource(R.string.stripe_paymentsheet_or_pay_with_card)
 ) {
-    PaymentsTheme {
-        Box(
-            contentAlignment = Alignment.Center,
+    Box(
```

Comment:
The only thing we're doing here is removing the `PaymentsTheme` the diff makes it look like I changed a lot more lol.